src/test/java/Utils/BrowserManager.java

```diff
@@ -0,0 +1,31 @@
+package Utils;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.firefox.FirefoxOptions;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+public class BrowserManager {
+
+	public static WebDriver startConfig(){
+
+		switch (System.getProperty("exeBrowser","chrome")) {
+		case "chrome":
+			WebDriverManager wdmC = WebDriverManager.chromedriver();
```

Comment:
what is C, use complete names. Avoid subjectivity

Reply:
summary expression in return